### ISSUE TITLE
Emacs integration requires some extra work

### ISSUE THREAD

**Issue by currentoor** [fulcrologic/fulcro-lein-template#6]
Emacs integration is broken out of the box. Results in the following warning and the repl does not work properly.
```
WARNING: CIDER's version (0.17.0-snapshot) does not match cider-nrepl's version (nil). Things will break!
```

The fix is to add these two dependencies to the `:cljs` profile in `project.clj`.
```
[cider/cider-nrepl "0.17.0-SNAPSHOT"]
[refactor-nrepl "2.4.0-SNAPSHOT"]
```

This fix was not immediately obvious to me, since we are using lein deps from shadow-cljs.

Should we do that conditionally, like ask if they want emacs/cider?

Or should we document this somewhere?

**Comment by awkay** [comment]
I don't use emacs. Feel free to fix it.

**Comment by awkay** [comment]
I don't think it needs to be conditional...what does it hurt to include them?
